extension/user/getUser.js

```diff
@@ -0,0 +1,23 @@
+const {EACCESS, ENOTFOUND, customError} = require('../error')
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {function} cb
+ */
+module.exports = (context, input, cb) => {
+  if (!context.meta.userId) {
+    return cb(customError(EACCESS, 'Unauthorized access'))
```

Comment:
The resulting message should be something readable for the user, just in case the frontend wants to show it.